src/dbus/mutter_display_config.rs

```diff
@@ -123,22 +291,11 @@ impl DisplayConfig {
 
                 let logical = output.logical.as_ref().unwrap();
 
-                let transform = match logical.transform {
-                    niri_ipc::Transform::Normal => 0,
-                    niri_ipc::Transform::_90 => 1,
-                    niri_ipc::Transform::_180 => 2,
-                    niri_ipc::Transform::_270 => 3,
-                    niri_ipc::Transform::Flipped => 4,
-                    niri_ipc::Transform::Flipped90 => 5,
-                    niri_ipc::Transform::Flipped180 => 6,
-                    niri_ipc::Transform::Flipped270 => 7,
-                };
-
                 let logical_monitor = LogicalMonitor {
                     x: logical.x,
                     y: logical.y,
                     scale: logical.scale,
-                    transform,
+                    transform: logical.transform as u32,
```

Comment:
I think it might be a good idea to leave this match as is, since it's converting into what is expected for the D-Bus API. And the `as u32` is whatever we happened to use in niri_ipc.